scripts/release.py

```diff
@@ -1,14 +1,14 @@
 #!/usr/bin/env python
```

Comment:
I wonder if we should just delete this file entirely. Since we have the release automation now, this would never get used and is likely to cause confusion.